test/e2e/network/netpol/probe.go

```diff
@@ -57,18 +54,17 @@ type ProbeJob struct {
 
 // ProbeJobResults packages the data for the results of a pod->pod connectivity probe
 type ProbeJobResults struct {
-	Job         *ProbeJob
-	IsConnected bool
-	Err         error
-	Command     string
+	Job  *ProbeJob
+	IsOK bool
```

Comment:
given that you didn't actually change the semantics, you might as well revert `IsOK` back to `IsConnected`